### PR TITLE
Upgrade step clarification and fixes

### PR DESCRIPTION
**Pull request by btat**

### CODE REVIEW

File: adoc/Release-Notes.adoc
Comment (satadruroy) [inline adoc/Release-Notes.adoc:120]: Maybe change the heading "Steps for a fresh install of SUSE Cloud Application Platform 1.5 HA:" to include "Steps for a fresh install of SUSE Cloud Application Platform 1.5  **in the default HA setup**". And then be explicit that specifying `config.HA=true` and `config.HA_strict=false` will default the sizes of the roles with the exception of mysql which will be set to 1.

File: adoc/Release-Notes.adoc
Comment (satadruroy) [inline adoc/Release-Notes.adoc:138]: Do we not need to grab the new secret and certificate values again right after we upgraded UAA from 2.17.1 to 2.18.0? I think the secrets-generator would generate new secrets and a corresponding `INTERNAL_CA_CERT`...?

**Review by satadruroy — CHANGES_REQUESTED** [review]
Generally, looks ok - minor changes requested.

**Comment by gaktive** [comment]
Someone more powerful than me in git-fu can see once we're comfortable with this, if we can merge this into PR 83 :)

**Comment by btat** [comment]
Changing this to WIP , please hold off on merging for now. Spoke with @satadruroy earlier and additional updates need to be made.

**Comment by mook-as** [comment]
> Someone more powerful than me in git-fu can see once we're comfortable with this, if we can merge this into PR 83 :)

Hit `Edit` to the right of the PR title for this PR, and change the base to `cap-151`? Not sure that's what you meant, so I didn't do it for you… the result would be something like:

```
--+----------------------------------------------------+--- (master) ---
   \                                                  /
    \----- (#84, upgrade-command-fixes) ---+         /
     \                                      \       /
      \____ (#83, cap-151) __________________\_____/
```

**Comment by gaktive** [comment]
@satadruroy once approved, merge into `cap-151` per @mook-as' recommendation. However, if the 1.5.1 material goes out first, then we need a different tack.

**Comment by satadruroy** [comment]
> @satadruroy once approved, merge into `cap-151` per @mook-as' recommendation. However, if the 1.5.1 material goes out first, then we need a different tack.

I'll let @btat confirm but the discussed changes are not big. But we also need the release notes page upgrade section (the beginning of known issues section) to point to the docs for the upgrade instructions?

**Comment by btat** [comment]
I think the changes being made here are only specific to 1.5 and shouldn't affect the 1.5.1 PR?

**Comment by gaktive** [comment]
@btat I just want to make sure I don't potentially lose any of your edits as I work on the 1.5.1 material simultaneously.

**Comment by btat** [comment]
This should be ready for review again. Will work on the corresponding updates on the docs side for this.

Also update the base to `cap-151`

File: adoc/Release-Notes.adoc
Comment (satadruroy) [inline adoc/Release-Notes.adoc]: Sorry, one other place where the wording needs to be fixed - "In order to deploy {product} 1.5 or upgrade from {product} 1.4.1 in an HA configuration, you will first need to ~~scale the `mysql` role down to 1 instance~~ to be able to migrate from `cf-mysql` to `PXC`." Maybe change this to a more accurate wording of "...first need to **start the mysql role with 1 instance**..."

**Review by satadruroy — CHANGES_REQUESTED** [review]
1 minor nit...

File: adoc/Release-Notes.adoc
Comment (btat) [reply adoc/Release-Notes.adoc]: Fixed in https://github.com/SUSE/cap-release-notes/pull/84/commits/5fae74ac5fff0c0455731356b5ae3459a764d44c

**Pull request merged**
